src/server/qgsserverogcapi.h

```diff
@@ -0,0 +1,203 @@
+/***************************************************************************
+  qgsserverogcapi.h - QgsServerOgcApi
+
+ ---------------------
+ begin                : 10.7.2019
+ copyright            : (C) 2019 by ale
+ email                : [your-email-here]
```

Comment:
```suggestion
 email                : ale@qgis.org
```

Reply:
```suggestion
 email                : elpaso at itopen dot it
```

Reply:
Done, tip: run a grep for  `[your-email-here]`  :)